librz/core/cconfig.c

```diff
@@ -3797,7 +3797,9 @@ RZ_API int rz_core_config_init(RzCore *core) {
 		"pe", "pilot", "srec", "w32run", "zip", "all", "none", NULL);
 	SETB("flirt.sig.deflate", false, "enables/disables FLIRT zlib compression when creating a signature file (available only for .sig files)");
 	SETI("flirt.node.optimize", RZ_FLIRT_NODE_OPTIMIZE_MAX, "FLIRT optimization option when creating a signature file (none: 0, normal: 1, smallest: 2)");
-	SETPREF("flirt.sigdb.path", "", "Rizin sigdb location on the filesystem.");
+	char *home_sigdb = rz_path_system(RZ_SIGDB);
+	SETPREF("flirt.sigdb.path", home_sigdb, "Rizin sigdb location on the filesystem.");
+	free(home_sigdb);
```

Comment:
This way one cannot install sigdb both in user-wide location and system-wide locations (e.g. if we provide a set of signatures, but someone else creates a new signature database that a user wants to install, he may want to install them just for his user).

Can we assume that when flirt.sigdb.path == NULL we use both `rz_path_system(RZ_SIGDB)` and `rz_path_home(RZ_SIGDB)`?

Reply:
maybe i can move the `rz_path_system(RZ_SIGDB)` within the function that parses the sigdb values and use this as user-defined location?

Reply:
Is it possible to load a sigdb from multiple directories and "merge" the results?

Reply:
well, is not implemented, but we could have that evar as the user db and the system db hardcoded.

Reply:
I was thinking about the possibility of a user to "overwrite" an existing signature (e.g. he wants  his own signature to take precedence over the system one). Is that possible right now?